recipes/configs/llama3_2_vision/11B_full_single_device_pretrained.yaml

```diff
@@ -0,0 +1,104 @@
+# Config for single device full finetuning in full_finetune_single_device.py
```

Comment:
What's the purpose of adding this? Just so that we support the non-instruct version of the model? I'm a bit confused cause I thought one big diff with instruct-tuned vs not is the extra trainable special tokens on the text size, which this PR doesn't address

Reply:
this is just for testing. I would remove it before the PR is ready

Reply:
regarding the special token, thats a question for @pbontrager . I am not sure.